### PR TITLE
instructions on how to uploade videos for SIG Node

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
/sig node

as we are getting more chairs, need to share instructions

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
@haircommander FYI to share the load in future

File: sig-node/CONTRIBUTING.md [hunk 1]
```diff
@@ -97,3 +97,17 @@ Running a local cluster
...
+
+### upload meeting recording to YouTube (for chairs)
+
+1. Download the video file from zoom account `sig-node-leads@kubernetes.io`
+2. Log in to YouTube under the `kubernetes-sig-node` account (use switch account)
+3. Upload video to YouTube
+   - Name should follow the template: `<name of the meeting> yyyymmdd` like `Kubernetes SIG Node 20240618`.
+   - Add a link to the meeting notes and agenda document to the meeting description
+   - Add video to k8s-owned YouTube playlist:
+     - Once video if uploaded, open the video page, click +Save button under the video, select the checkbox `SIG Node Meetings`.
```
Comment (haircommander) [inline sig-node/CONTRIBUTING.md:111]: s/if/is ?

Comment (SergeyKanzhelev) [reply sig-node/CONTRIBUTING.md:111]:
```suggestion
     - Once video is uploaded, open the video page, click +Save button under the video, select the checkbox `SIG Node Meetings`.
```

**Comment by haircommander** [comment]
/lgtm

Just followed the instructions and it worked :)

**Pull request merged**